### PR TITLE
Updated README

### PR DESCRIPTION
**Pull request by kraklin**
To be honest, I have no idea what to write to README. Will the example be enough? I don't know

### CODE REVIEW

**Review by rofrol — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,10 +1,38 @@
 # Cmd.Extra
 
 Experimental package with helpful functions for working with Cmd.
-Note that this API is experimental and may go through more iterations.
+
+This package adds some functions that helps you with constructing of `Cmd msg` from given `msg` and theirs manipulation.
+
+```
+    type Msg
+        = DoNothing
```
Comment (turboMaCk) [inline README.md]: this is usually called `NoOp`

File: README.md [hunk 2]
```diff
@@ -1,10 +1,38 @@
...
+        | Disarm
+        | Fire
+        | FireRockets
+
+    update : Msg -> Model -> ( Model, Cmd Msg )
+    update msg model =
+        case msg of
+            DoNothing ->
+                Cmd.Extra.pure
```
Comment (turboMaCk) [inline README.md]: this is not right. pure expects model as an argument

**Review by turboMaCk — CHANGES_REQUESTED** [review]
see comments

File: README.md (hunk 2 above)
Comment (kraklin) [reply README.md]: 🤦‍♂️

File: README.md [hunk 3]
```diff
@@ -1,10 +1,38 @@
 # Cmd.Extra
 
 Experimental package with helpful functions for working with Cmd.
-Note that this API is experimental and may go through more iterations.
+
+This package adds some functions that helps you with constructing of `Cmd msg` from given `msg` and theirs manipulation.
```
Comment (turboMaCk) [inline README.md]: in fact manipulation is not about Cmd but about pair `(a, Cmd b)`

File: README.md [hunk 4]
```diff
@@ -2,11 +2,11 @@
 
 Experimental package with helpful functions for working with Cmd.
 
-This package adds some functions that helps you with constructing of `Cmd msg` from given `msg` and theirs manipulation.
+This package adds some functions that helps you with constructing of `(model, Cmd msg)` pair from given `model` and `msg` and theirs manipulation.
```
Comment (turboMaCk) [inline README.md]: this is also not acurate. perform attempt and maybe are not working with pair.

there are 2 things that this package provides:

1. constructors for Cmds
2. way to compose (model, Cmd msg) pairs in pipelines.

File: README.md [hunk 5]
```diff
@@ -2,7 +2,7 @@
 
 Experimental package with helpful functions for working with Cmd.
 
-This package adds some functions that helps you with constructing of `(model, Cmd msg)` pair from given `model` and `msg` and theirs manipulation.
+This package can help you with construction of `Cmd msg` and composition of pair `(model, Cmd msg)` that can be used to streamline your flow of actions in `update`.
```
Comment (turboMaCk) [inline README.md]: great!!!

**Review by turboMaCk — APPROVED** [review]
LGTM

**Comment by kraklin** [comment]
Can you publish it @turboMaCk ? I'm not sure if I'm able to do that.

**Pull request merged**